spec/unit/installer/sandbox_dir_cleaner_spec.rb

```diff
@@ -32,15 +36,25 @@ module Pod
       random_pod_public_headers = @sandbox.public_headers.root.join('PublicPod')
       banana_lib_public_headers = @sandbox.public_headers.root.join('BananaLib')
       banana_lib_private_headers = @pod_target.build_headers.root.join('BananaLib')
+      coconut_lib_public_headers = @sandbox.public_headers.root.join('CoconutLibModule')
+      coconut_lib_private_headers = @pod_target.build_headers.root.join('CoconutLibModule')
       FileUtils.mkdir_p(random_pod_private_headers)
       FileUtils.mkdir_p(random_pod_public_headers)
       FileUtils.mkdir_p(banana_lib_public_headers)
       FileUtils.mkdir_p(banana_lib_private_headers)
+      FileUtils.mkdir_p(coconut_lib_public_headers)
+      FileUtils.mkdir_p(coconut_lib_private_headers)
       @cleaner.expects(:remove_dir).with(random_pod_private_headers)
       @cleaner.expects(:remove_dir).with(random_pod_public_headers)
+      @cleaner.expects(:remove_dir).with(coconut_lib_public_headers).never
+      @cleaner.expects(:remove_dir).with(coconut_lib_private_headers)
       @cleaner.clean!
-      banana_lib_public_headers.should.exist?
```

Comment:
what about these assertions?

Reply:
since `:remove_dir` is stubbed above, these assertions will always pass. they aren't testing anything